### PR TITLE
Made changes based on recommendations by Neil and also fixed minor abstract rendering issues in a few other papers

### PR DESCRIPTION
**Pull request by sgunasekar**
I made changes recommended by Neil.

### CODE REVIEW

File: _posts/2021-07-21-braverman21a.md [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-  closely related to bracketing numbers in statistics and to hyperplane cuttings in
-  discrete geometry, and allows for a compressed approximate representation of every
-  halfspace. We complement our upper bound result by an almost matching $\tilde \Omega(d\log
-  n)$ lower bound on the communication complexity of any such protocol.  Since the
-  distributed halfspace learning problem is closely related to the convex set disjointness
-  problem in communication complexity and the problem of distributed linear programming
-  in distributed optimization, we also derive upper and lower bounds of $\tilde O(d^2\log
-  n)$ and $\tilde{\Omega}(d\log n)$ on the communication complexity of both of these
-  basic problems.'
+abstract: '{\em Distributed learning} protocols are designed to train on distributed data without gathering it all on a single centralized machine,
```
Comment (lawrennd) [inline _posts/2021-07-21-braverman21a.md]: We can use latex foe maths in abstract, but need html for itlaics.

File: _posts/2021-07-21-braverman21a.md [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
+thus contributing to the efficiency of the system and enhancing its privacy. 
+We study a central problem in distributed learning, called {\it distributed learning of halfspaces}: 
+let $U \subseteq \mathbb{R}^d$ be a known domain of size $n$ and let $h:\mathbb{R}^d\to \mathbb{R}$ be an unknown target affine function.\footnote{In practice, the domain $U$ is defined implicitly by the representation of $d$-dimensional vectors which is used in the protocol.}
+A set of {\it examples} $\{(u,b)\}$ is distributed between several parties, where~$u \in U$ is a point and $b = \mathsf{sign}(h(u)) \in \{\pm 1\}$ is its label. 
+The parties' goal is to agree on a classifier~$f: U\to\{\pm 1\}$ such that~$f(u)=b$ for every input example~$(u,b)$.
+
+We design a protocol for the distributed halfspace learning problem in the two-party setting, communicating only $\tilde O(d\log n)$ bits.  
+To this end, we introduce a new tool called {\em halfspace containers}, that is closely related to {\em bracketing numbers} in statistics and to {\it hyperplane cuttings} in discrete geometry,
```
Comment (lawrennd) [inline _posts/2021-07-21-braverman21a.md]: HTML needed here.

File: _posts/2021-07-21-lee21a.md [hunk 3]
```diff
@@ -1,27 +1,12 @@
...
-  samplers with nontrivial gradient query complexity were previously known. \item
-  For densities with condition number $\kappa$, we give an algorithm obtaining mixing
-  time $O(\kappa d \log^2\tfrac{\kappa d}{\epsilon})$, improving \cite{LeeST20} by
-  a logarithmic factor with a significantly simpler analysis. We also show a zeroth-order
-  algorithm attains the same query complexity.  \end{itemize}'
+abstract: 'We give algorithms for sampling several structured logconcave families to high accuracy. We further develop a reduction framework, inspired by proximal point methods in convex optimization, which bootstraps samplers for regularized densities to generically improve dependences on problem conditioning $\kappa$ from polynomial to linear. A key ingredient in our framework is the notion of a ``restricted Gaussian oracle'' (RGO) for $g: \mathbb{R}^d \rightarrow \mathbb{R}$, which is a sampler for distributions whose negative log-likelihood sums a quadratic (in a multiple of the identity) and $g$. By combining our reduction framework with our new samplers, we obtain the following bounds for sampling structured distributions to total variation distance $\eps$.
```
Comment (lawrennd) [inline _posts/2021-07-21-lee21a.md]: Latex quotes in this abstract, use normal quotes

**Review by lawrennd — CHANGES_REQUESTED** [review]
Wow Suriya, great work on tidying up the maths. There's a couple of spots where we need html tags instead of latex though.

**Comment by sgunasekar** [comment]
I fixed a couple more abstracts with math issues and the html tag issues raised by Neil.

**Review by lawrennd — APPROVED** [review]
Thanks so much for your patience on these!

**Pull request merged**